openstack/sharedfilesystems/v2/shares/requests.go

```diff
@@ -174,10 +174,17 @@ func Get(client *gophercloud.ServiceClient, id string) (r GetResult) {
 	return
 }
 
-// GetExportLocations will get shareID's export locations.
-// Client must have Microversion set; minimum supported microversion for GetExportLocations is 2.14.
-func GetExportLocations(client *gophercloud.ServiceClient, id string) (r GetExportLocationsResult) {
-	_, r.Err = client.Get(getExportLocationsURL(client, id), &r.Body, nil)
+// ListExportLocations will list shareID's export locations.
+// Client must have Microversion set; minimum supported microversion for ListExportLocations is 2.9.
+func ListExportLocations(client *gophercloud.ServiceClient, id string) (r ListExportLocationsResult) {
```

Comment:
I'm OK with this change and will make note of it in the CHANGELOG. Originally, I think `Get` was found acceptable since there wasn't any paging.